README.md

```diff
@@ -58,6 +58,25 @@ installMediaQueryWatcher(`(min-width: 600px)`, (matches) => {
 });
 ```
 
+### `axe-report.js`
+This is an [axe-core](https://github.com/dequelabs/axe-core) reporter that returns an
+Error containing every a11y violation for an element. Use this if you want to
+include `axe-core` in automated Mocha tests, etc.
+
+Example (in a Mocha test):
+```js
+import '../node_modules/axe-core/axe.min.js';
+import { axeReport } from '../node_modules/@polymer/pwa-helpers/connect-mixin.js';
```

Comment:
Should be axe-report.js

Reply:
all the other examples use `@polymer` for now (since we haven't decided where this will get published on npm yet)

Reply:
Sounds good. Maybe just `s/connect-mixin.js/axe-report.js`

Reply:
oh yeah that's defo wrong 😂